layout/pulse/layer-menu-native/component.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import findIndex from 'lodash/findIndex';
+
+import Icon from 'components/ui/Icon';
+
+class LayerMenuNativeComponent extends PureComponent {
+  render() {
+    const { layers, triggerClick } = this.props;
+    return (
+      <div className="c-layer-menu-native">
+        <select
+          onChange={(event) => {
+            const { value } = event.target;
+            if (value === -1) {
+              triggerClick(null);
+            } else {
+              layers.forEach((g) => {
+                const index = findIndex(g.layers, layer => event.target.value === layer.label);
```

Comment:
Do you know that we have a native findIndex => https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Array/findIndex 

It's true that we need a polyfill. But we are loading it https://github.com/resource-watch/resource-watch/blame/feature/pulse-responsive/layout/head/app.js#L158